presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,76 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving user-defined
+functions with a persistent storage, allowing users to create, alter,
+and delete functions, as well as to reference them within queries.
+
+Function Namespace
+------------------
+
+A function namespace is a catalog-schema name (i.e. ``example.test``)
+in which each function can be uniquely identified by its signature,
+that is, its name and parameter type list.
+
+The function namespace, such as ``example.test``, is not a full-fledged
+schema as it does not support storing tables and views, but only
+functions.
+
+Each instance of function namespace manager manages a catalog. Creating
```

Comment:
What is a "function namespace manager"? Define it first.

Reply:
It purpose and functionality is already defined in the first paragraph. Here, I'm expanding the description.